src/klein/_app.py

```diff
@@ -268,7 +291,10 @@ def index(request):
 
         @named("router for '" + url + "'")
         def deco(f: KleinRoute) -> KleinRoute:
-            kwargs.setdefault("endpoint", f.__name__)
+            kwargs.setdefault(
+                "endpoint",
+                f.__name__,  # type: ignore[attr-defined]
```

Comment:
To remove the `ignore` here would mean adding the `__name__` attribute to `KleinRoute`.

I think we should probably do that, but I'm holding off on that until I can get the type hints for `Klein.route` to work.